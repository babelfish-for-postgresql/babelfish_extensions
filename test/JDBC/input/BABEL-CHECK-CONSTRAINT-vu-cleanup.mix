-- tsql
DROP TABLE [dbo].[alter_table_check_constraint]
GO

DROP TABLE check_constraint
GO

DROP TABLE create_check_constraint
GO

-- psql
DROP TABLE master_dbo.tbl_creation_should_succeed;
GO
