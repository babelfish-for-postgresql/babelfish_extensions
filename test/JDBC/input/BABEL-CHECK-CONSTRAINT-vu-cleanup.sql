DROP TABLE [dbo].[alter_table_check_constraint]
GO

DROP TABLE check_constraint
GO

DROP TABLE create_check_constraint
GO