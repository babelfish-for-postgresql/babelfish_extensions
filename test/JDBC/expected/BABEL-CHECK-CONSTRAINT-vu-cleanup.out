-- tsql
DROP TABLE [dbo].[alter_table_check_constraint]
GO

DROP TABLE check_constraint
GO

DROP TABLE create_check_constraint
GO

-- psql
DROP TABLE tbl_creation_should_succeed;
GO
